### ISSUE TITLE
proxmox lxc container

### ISSUE THREAD

**Issue by panayotovip** [guydavis/machinaris#89]
has the development of a version for proxmox lxc container been considered ?

**Comment by guydavis** [comment]
No, but sounds interesting.  Please provide any further details here.  Any help would be appreciated!

**Comment by cmbcbe** [comment]
You can use docker inside LXC, just use unprivilegied and nesting feature. Problem with lxc is that the application is listed inside the process of root computer, thus can be detectable by provider and banned for crypto mining while inside a true VM, this is invisible...
